### PR TITLE
Feature add annotate excel guide

### PR DESCRIPTION
**Pull request by Etschbeijer**
# Thank you for contributing to the DataPLANT Knowledge Base :rocket:

## Please make sure that

- [x] your pull request has a **good title**,
- [x] you add a short description below to summarize your changes,
- [x] the contents and images you add, are allowed to be shared publicly and - if applicable - reference the source properly,
- [x] you read the [contribution guide](https://github.com/nfdi4plants/nfdi4plants.knowledgebase/blob/main/CONTRIBUTING.md),
- [x] you tested your changes locally via `npm run dev` and `npm run build`

If you are unsure, you can also label your PR as draft.

## Description
Add documentation for swate excel
Updated several existing swate pages

### CODE REVIEW

File: src/content/docs/guides/annotate-in-excel.mdx [hunk 1]
```diff
@@ -2,11 +2,30 @@
...
 
-Lorem ipsum
\ No newline at end of file
+import { Steps } from '@astrojs/starlight/components';
+
+With the help of an [Excel-Add-In](https://learn.microsoft.com/de-de/office/dev/add-ins/excel/excel-add-ins-overview) you can manage your data, utilizing the [ARC](https://nfdi4plants.github.io/nfdi4plants.knowledgebase/core-concepts/arc/) standard, directly in excel, without the necessity to upload it anywhere. 
+The functionality the excel-add-in provides are similar to [Swate](https://nfdi4plants.github.io/nfdi4plants.knowledgebase/swate/) and the differences will be explained in the different sections of swate.
+The excel-add-in will soon be published in the official [Microsoft Excel-Add-Ins Store](https://appsource.microsoft.com/en-us/marketplace/apps?product=excel). 
+
+## Local installation
```
Comment (Freymaurer) [inline src/content/docs/guides/annotate-in-excel.mdx:19]: must mention online installation as currently recommended workflow. Please use starlight tabs for this

File: src/content/docs/swate/add-building-blocks.mdx [hunk 2]
```diff
@@ -4,11 +4,24 @@ lastUpdated: 2024-04-26
 authors: 
   - kevin-frey
   - stella-eggels
+  - patrick-blume
 ---
 
+import { Steps } from '@astrojs/starlight/components';
+import { Tabs, TabItem } from '@astrojs/starlight/components';
+
+<Tabs syncKey="host">
```
Comment (Freymaurer) [inline src/content/docs/swate/add-building-blocks.mdx:13]: rename syncKey to more descriptive "swateHost"

File: src/content/docs/swate/add-building-blocks.mdx [hunk 3]
```diff
@@ -4,11 +4,24 @@ lastUpdated: 2024-04-26
...
+<TabItem label="Browser">
```
Comment (Freymaurer) [inline src/content/docs/swate/add-building-blocks.mdx:14]: Add icons to TabItem

https://starlight.astro.build/components/tabs/#add-icons-to-tabs

File: src/content/docs/swate/add-building-blocks.mdx [hunk 4]
```diff
@@ -42,7 +55,34 @@ import { Steps } from '@astrojs/starlight/components';
 
     If you cannot find a fitting term, you can write an issue to request an ontology term [here](https://github.com/nfdi4plants/nfdi4plants_ontology/issues/new/choose) **OR** use free text input.
```
Comment (Freymaurer) [inline src/content/docs/swate/add-building-blocks.mdx:56]: make this maybe a 

:::note 

:::

File: src/content/docs/swate/annotation-table.mdx [hunk 5]
```diff
@@ -4,24 +4,66 @@ lastUpdated: 2024-04-25
...
+When you have an empty excel sheet or one hat already contains an annotation table, you can click on the "Create Annotation Table" icon of the navbar of the sidebar.
+When the excel sheet contains no annotation table, a new annotation table will be created in the active excel sheet.
+When an annotation table already exists in the active excel sheet, a new excel sheet will be created, become active and a new annotation table will be created in that sheet.
+
+The new annotation table will be created in the cell that was selected, when you the "Create Annotation Table" icon was clicked.
+The newly created annotation table has a size of 1 Input [Source Name] column and row.
+
+![Image showing add new file option in Swate](@images/swate/swate-excel-add-annotation-table.png)
+
+## Top level metadata handling
```
Comment (Freymaurer) [inline src/content/docs/swate/annotation-table.mdx:52]: make this a new file

**Review by Freymaurer — CHANGES_REQUESTED** [review]
Some minor changes then good-to-go

File: src/content/docs/swate/add-building-blocks.mdx (hunk 3 above)
Comment (Etschbeijer) [reply src/content/docs/swate/add-building-blocks.mdx:14]: ignored

File: src/content/docs/swate/annotation-table.mdx (hunk 5 above)
Comment (Etschbeijer) [reply src/content/docs/swate/annotation-table.mdx]: fixed

File: src/content/docs/swate/add-building-blocks.mdx (hunk 4 above)
Comment (Etschbeijer) [reply src/content/docs/swate/add-building-blocks.mdx:57]: fixed

File: src/content/docs/swate/add-building-blocks.mdx (hunk 2 above)
Comment (Etschbeijer) [reply src/content/docs/swate/add-building-blocks.mdx]: fixed

File: src/content/docs/guides/annotate-in-excel.mdx (hunk 1 above)
Comment (Etschbeijer) [reply src/content/docs/guides/annotate-in-excel.mdx:54]: fixed

File: src/assets/images/swate/swate-browser-excel-add-in-additional-ones.png
Comment (Freymaurer) [inline src/assets/images/swate/swate-browser-excel-add-in-additional-ones.png:1]: make this image WAY smaller. So much dead space

File: src/content/docs/guides/annotate-in-excel.mdx [hunk 6]
```diff
@@ -11,21 +11,57 @@ sidebar:
...
 import { Steps } from '@astrojs/starlight/components';
+import { Tabs, TabItem } from '@astrojs/starlight/components';
 
 With the help of an [Excel-Add-In](https://learn.microsoft.com/de-de/office/dev/add-ins/excel/excel-add-ins-overview) you can manage your data, utilizing the [ARC](https://nfdi4plants.github.io/nfdi4plants.knowledgebase/core-concepts/arc/) standard, directly in excel, without the necessity to upload it anywhere. 
 The functionality the excel-add-in provides are similar to [Swate](https://nfdi4plants.github.io/nfdi4plants.knowledgebase/swate/) and the differences will be explained in the different sections of swate.
-The excel-add-in will soon be published in the official [Microsoft Excel-Add-Ins Store](https://appsource.microsoft.com/en-us/marketplace/apps?product=excel). 
+The excel-add-in will soon be published in the official [Microsoft Excel-Add-Ins Store](https://appsource.microsoft.com/en-us/marketplace/apps?product=excel).
+Currently, we recommend the installation using the [excel-browser version provided by microsoft](https://excel.cloud.microsoft/?wdOrigin=MARKETING.FREE.GO-TO-EXCEL) because the set up is much easier than with a local excel version.
+
```
Comment (Freymaurer) [inline src/content/docs/guides/annotate-in-excel.mdx:20]: Add "installation" header here and remove headers in tabs.

Indent images to the right so they can be included in steps and do not restart the steps component after each image. Use one steps component and indent the content to fit it into the steps component

File: src/assets/images/swate/swate-browser-excel-add-in-additional-ones.png
Comment (Etschbeijer) [reply src/assets/images/swate/swate-browser-excel-add-in-additional-ones.png:1]: fixed

File: src/content/docs/guides/annotate-in-excel.mdx (hunk 6 above)
Comment (Etschbeijer) [reply src/content/docs/guides/annotate-in-excel.mdx:22]: fixed

**Review by Freymaurer — CHANGES_REQUESTED** [review]

**Review by Freymaurer — APPROVED** [review]

**Pull request merged**